packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -129,6 +146,24 @@ Resources:
                   Ref: AWS::AccountId
                 kms:ViaService:
                   - Fn::Sub: s3.${AWS::Region}.amazonaws.com
+          - Action:
```

Comment:
I think ECR also needs to call kms:RetireGrant if the repository is ever deleted.

Reply:
The kms KeyPolicy allows `kms:Create*` and `kms:Delete*` for the account. https://github.com/aws/aws-cdk/blob/1197f68f6dbef2b15b6c1595687a4a5497f3e97e/packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml#L112-L130

The encrypt and decrypt actions are specific for ecr. So we are fine here.

Reply:
`kms:Delete*` covers `kms:RetireGrant`?

Reply:
True! Got confused while thinking about it ^^ Then we should definitely add `kms:RetireGrant`